go.mod

```diff
@@ -26,6 +26,6 @@ require (
 )
 
 replace (
-	github.com/kata-containers/kata-containers/src/runtime => ../kata-containers/src/runtime
+	github.com/kata-containers/kata-containers/src/runtime => github.com/yoheiueda/kata-containers/src/runtime v0.0.0-20220429130351-22a2aa8a867d
```

Comment:
I think it's better to avoid this change for now
Instead let's update the dev doc - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/docs/DEVELOPMENT.md#clone-source-code

Reply:
This avoids explicit reference and keep the developer flow intact. Otherwise for any change we need to explicitly modify the go.mod.
Sometime back I had created a dev guide which (my bad) is still sitting on my private google doc. Here is a link to the same - https://hackmd.io/nBPF7ounQoapOZTelkmXKA
Please see if it helps. Will create a PR once you confirm

Reply:
Yes, the `replace` statement is intentionally used to avoid explicit reference to my personal repository. When the changes are merged to the upstream kata containers repo, we can update `go.mod` not to use `replace`.

I agree that the current setup procedure is somewhat error prone due to this `replace` statement in go.mod. If there is  a better way to refer the patched kata containers repo, we can switch to it.